.eslintrc.json

```diff
@@ -3,6 +3,7 @@
         "es6": true,
         "node": true
     },
+    "parser": "babel-eslint",
```

Comment:
It was a architecture decision not to use babel on the server side code, we should avoid linting with this parser.